wrapper/lib/wrapper.js

```diff
@@ -52,7 +52,7 @@ async function compileProviderIsReady () {
    *
    * @since 0.x.x
    */
-  ; (function (plugin) {
+  (function (plugin) {
```

Comment:
Did the linter recommend removing this? It could very easily lead to bugs in the future...